core/services/log/broadcaster.go

```diff
@@ -48,12 +48,16 @@ type (
 
 		IsConnected() bool
 		Register(listener Listener, opts ListenerOpts) (unsubscribe func())
-		BackfillBlockNumber() null.Int64
 
-		TrackedAddressesCount() uint32
-		// DB interactions
 		WasAlreadyConsumed(db *gorm.DB, lb Broadcast) (bool, error)
 		MarkConsumed(db *gorm.DB, lb Broadcast) error
+		// NOTE: WasAlreadyConsumed and MarkConsumed MUST be used within a single goroutine in order for WasAlreadyConsumed to be accurate
```

Comment:
I would have hoped this was obvious... :/

Reply:
i think it was an issue in one of subscribers at some point